src/Xamarin.Android.Build.Tasks/Xamarin.Android.Bindings.targets

```diff
@@ -294,7 +294,7 @@ Copyright (C) 2012 Xamarin Inc. All rights reserved.
       AndroidNdkDirectory="$(_AndroidNdkDirectory)"
       Assemblies="@(ReferencePath);@(ReferenceDependencyPaths)"
       CacheFile="$(IntermediateOutputPath)resourcepaths.cache"
-      Condition=" '$(DesignTimeBuild)' != 'true' "
+      Condition=" '$(DesignTimeBuild)' == 'false' Or '$(DesignTimeBuild)' == '' "
```

Comment:
...and here we're checking for empty. Would it be better if we *ensure* it was either True or False?

Also, why does `!= 'true'` not work?